lib/inject_remake.rb

```diff
@@ -0,0 +1,7 @@
+class Array
+
+  def inject_remake(total = self.shift)
+      self.each {|element| total = yield total, element}
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Space between { and | missing.<br>Space missing inside }.